website/docs/fatalwarnings.md

```diff
@@ -10,20 +10,29 @@ fatalwarnings { "warnings" }
 
 For Visual Studio, the MSC warning number should be used to specify the warning. On other compilers, the warning should be identified by name.
 
-In addition, Premake provides two special values to turn on all compiler and linker warnings.
+In addition, Premake provides a special value to turn on all compiler warnings.
 
 | Value   | Description                   |
 -------------------------------------------
-| Compile | Treat all compiler warnings as errors |
-| Link    | Treat all linker warnings as errors   |
+| All | Treat all compiler warnings as errors |
 
 ### Applies To ###
 
 Project configurations.
 
 ### Availability ###
 
-Premake 5.0 or later. `Compile` and `Link` special values available since Premake 5.0-beta4 or later.
+Premake 5.0 or later. Special value `All` available since Premake 5.0-beta5 or later.
+
+### Examples ###
+
+```lua
+filter { "toolset:msc" }
+	fatalwarnings { "4035" }
+
+filter { "toolset:clang" }
+	fatalwarnings { "-Wreturn-type" }
```

Comment:
```suggestion
filter { "toolset:msc" }
	fatalwarnings { "4035" } -- 'function': no return value

filter { "toolset:clang" }
	fatalwarnings { "-Wreturn-type" }

filter {}
```
Closing scope, someone even would have preferred extra `filter {}` before opening the other one ;-)
I also add comment as number is not self explanatory,
even if here, with clang part, we might expect same warning.

Reply:
Also good suggestions, same comments as the other one apply :)